dvol_python/test_plugin.py

```diff
@@ -83,7 +83,6 @@ def dvol_list_includes_memorydiskserver():
                 raise Exception("volume never showed up in result %s" % (result,))
         try_until(dvol_list_includes_memorydiskserver)
 
-    @skip_if_go_version
     def test_docker_run_dvol_container_show_up_in_list_output(self):
         container = "fancy"
         def cleanup():
```

Comment:
This test and the one beneath it has potential bugs in the cleanup code. If the first run call raises, the second run call will not, meaning memorydiskserver is not removed. It has been fixed in the other tests, please could you update these ones as well?